module/move/wca/src/ca/grammar/dictionary.rs

```diff
@@ -1,10 +1,14 @@
 pub( crate ) mod private
 {
+  use std::cmp::Ordering;
   use crate::*;
 
   use { Command };
-  use std::collections::HashMap;
+  use std::collections::BTreeMap;
+  use std::fmt::Display;
   use former::Former;
+  use crate::ca::aggregator::private::Order;
```

Comment:
This seems to be not in line with the [convention](https://github.com/obox-systems/conventions/blob/master/code_style.md#importing-local-entities)